Dockerfile

```diff
@@ -101,9 +101,11 @@ COPY --from=builder /usr/local/lib/erlang/lib/p1_utils_iconv-* /usr/local/lib/er
 WORKDIR /opt/chat-service
 COPY --from=builder /opt/chat-service .
 COPY --from=builder /tmp/chat-service/scripts scripts
+WORKDIR /home/ejabberd/.ejabberd-modules
+COPY --from=builder /home/ejabberd/.ejabberd-modules .
 
-USER ejabberd
 WORKDIR /home/ejabberd
+USER ejabberd
```

Comment:
Order doesn't actually matter here. Must have changed it up when I was figuring out where to put the modules. Going to leave this as is, since it's going to take another 15 minutes to build the image and republish the image if I change it back.